docs/user-guide/cli-install-configure-zosmf-security.md

```diff
@@ -0,0 +1,59 @@
+| z/OSMF workflow services | Endpoints that begin with: `/zosmf/workflow/` | Cloud provisioning for development environments. | <ul><li>[z/OSMF workflow services](https://www.ibm.com/docs/en/zos/2.5.0?topic=services-zosmf-workflow)</li></ul> |
+
+## Configuring z/OS console REST interface
+
+Review the following recommendations for configuring the security for z/OS console REST services:
+
+- Add the COMMON_TSO statement to the IZUPRMxx parmlib member to customize the z/OSMF options for the logon procedure.
+- Define a value of at least 50000 KB as the size of the address space for the user's logon procedure. To help you prevent system memory exception errors from occurring, confirm that this value is acceptable in your environment.
+- Ensure that the members in your z/OSMF user security group can issue TSO and CONSOLE commands. IBM provides RACF statements for user group IZUUSER. To prevent all z/OSMF users from issuing TSO and CONSOLE commands, you can create more z/OSMF user groups for more granular security.
+- Ensure that the OPERCMD class is active and that your MVS commands are protected. MVS commands include, but are not limited to, the MVS and MVS.MCSOPER resource prefixes.
+- Ensure that the z/OSMF user security groups can access (authorized) the logon procedure name and account number that is specified in the COMMON_TSO statement.
+- Define a TSO segment for all the z/OSMF users.
+
+## Configuring z/OS data set and file REST interface
+
+Review the following recommendations for configuring the z/OS security for data set and file REST services:
+
+- Add the COMMON_TSO statement to the RESTAPI_FILE parmlib member to customize the z/OSMF options for the logon procedure.
+- Define a value of at least 65536 KB as the size of the address space for the user's logon procedure. To help you prevent system memory exception errors from occurring, confirm that this value is acceptable in your environment.
+- Authorize z/OSMF user groups and the z/OSMF server to CEA TSO/E address space services.
+- Ensure that the z/OSMF user security groups can access (authorized) the logon procedure name and account number that is specified on the COMMON_TSO statement.
+- Define a TSO segment for all the z/OSMF users.
+- Define at least 20971520 KB (20 MB) the IPCMSGQBYTES option of your parmlib member named BPXPRMxx. IBM recommends this value to let TSO and z/OSMF communicate using z/OS USS interprocess communications.
+
+## Configuring z/OSMF plug-in security
+Ensure that you implement all the required security for the plug-ins. For more information, see [Setting up security for the z/OSMF plug-ins](https://www.ibm.com/docs/en/zos/2.2.0?topic=configuration-setting-up-security-zosmf-plug-ins) in the IBM Documentation.
```

Comment:
@adam-wolfe @zFernand0 -- I couldn't five the v2.5 equivalent for the v2.2 article "Setting up security for the z/OSMF plug-ins" in the IBM Docs. Could one of you help me find a replacement?

Reply:
It got a lot more detailed and broken down for each service:
https://www.ibm.com/docs/en/zos/2.5.0?topic=guide-security-structures-zosmf